salt/salt-auto-update.py

```diff
@@ -64,13 +63,15 @@ def main():
         print("  job id", job_id)
         state.record_update(minion_id, job_id)
 
-
-def load_file_lines(filename):
-    try:
-        return set(x.strip() for x in open(filename))
-    except IOError:
-        return set()
-
+def get_blacklist():
+    result = subprocess.check_output(["salt", "-N", "blacklist", "--preview-target"])
+    lines = result.split(b"\n")
+    deviceSet = set()
+    for i in lines:
```

Comment:
`i` is typically reserved for integer loop counters or indexes. How about `for line in lines:`?